.travis.yml

```diff
@@ -0,0 +1,61 @@
+# <~> Travis integration file.
```

Comment:
Don't prefix comments with decorative elements. (The fact that this is *your* contribution is tracked by Git just fine.)